### PR TITLE
chore: re-enable disabled test since quotas already increased

### LINKED ISSUE #947: [Dialogflow] Reenable samples tests, they keep reaching quota.
Even after #936, quota is still being reached. I can't reproduce this in local. Will disable samples again until I hava enough time to properly look into it.

### PR DESCRIPTION
**Pull request by munkhuushmgl**
Fixes #947

### CODE REVIEW

**Comment by amanda-tarafa** [comment]
They failed presubmit on Windows, I think because of the dependencies are old. Can you update the dependecies of the samples to netstandard2.1 and of the tests to netcoreapp3.1 and make sure that the libraries are the newest?

**Comment by munkhuushmgl** [comment]
> They failed presubmit on Windows, I think because of the dependencies are old. Can you update the dependecies of the samples to netstandard2.1 and of the tests to netcoreapp3.1 and make sure that the libraries are the newest?

Ok will do

**Comment by munkhuushmgl** [comment]
@amanda-tarafa PTaL looks like something outside of my changes failing

File: dialogflow/api/Test/DialogflowSampleTests.csproj
Comment (amanda-tarafa) [inline dialogflow/api/Test/DialogflowSampleTests.csproj:7]: Remove this package reference. You already get it when you reference the sample project.

**Review by amanda-tarafa — CHANGES_REQUESTED** [review]
Just a minor detail.
Otherwise seems to have passed. Once merged I'll keep a close eye on it and see if we reach quota again.

**Review by amanda-tarafa — APPROVED** [review]
LGTM. Will merge after running tests again.

**Pull request merged**
